highfive/highfive.hpp build issue with libkea with conda-forge builds

@gillins  GDAL Windows conda-based builds just started to fail with:
```
C:\PROGRA~1\MICROS~2\18\ENTERP~1\VC\Tools\MSVC\1451~1.362\bin\Hostx64\x64\cl.exe  /nologo /TP -DGDAL_COMPILATION -DHAVE_AVX_AT_COMPILE_TIME -DHAVE_OPENMP -DHAVE_SSE_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DNOMINMAX -DPLUGIN_FILENAME=\"gdal_KEA.dll\" -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_MBCS -ID:\a\gdal\gdal\apps -ID:\a\gdal\gdal\alg -ID:\a\gdal\gdal\gcore -ID:\a\gdal\gdal\gcore\multidim -ID:\a\gdal\gdal\build\gcore -ID:\a\gdal\gdal\port -ID:\a\gdal\gdal\build\port -ID:\a\gdal\gdal\ogr -ID:\a\gdal\gdal\ogr\ogrsf_frmts -ID:\a\gdal\gdal\frmts -ID:\a\gdal\gdal\frmts\mem -IC:\Users\runneradmin\miniconda3\envs\gdalenv\Library\include /WX /O2 /Ob2 /DNDEBUG -std:c++17 -MD /EHsc /Zc:__cplusplus /MP -openmp /showIncludes /Fofrmts\kea\CMakeFiles\gdal_KEA_core.dir\Unity\unity_0_cxx.cxx.obj /Fdfrmts\kea\CMakeFiles\gdal_KEA_core.dir\ /FS -c D:\a\gdal\gdal\build\frmts\kea\CMakeFiles\gdal_KEA_core.dir\Unity\unity_0_cxx.cxx
C:\Users\runneradmin\miniconda3\envs\gdalenv\Library\include\libkea/KEACommon.h(35): fatal error C1083: Cannot open include file: 'highfive/highfive.hpp': No such file or directory
```

See https://github.com/OSGeo/gdal/actions/runs/28103178245/job/83210256259?pr=14856

--- a/.github/workflows/cmake_builds.yml
+++ b/.github/workflows/cmake_builds.yml
@@ -438,8 +438,8 @@ jobs:
       shell: bash -l {0}
       run: |
         conda install --yes --quiet curl libiconv icu python=3.12 pip swig numpy "pytest!=9.1.0" pytest-env pytest-benchmark filelock zlib lxml jsonschema setuptools
-        conda install --yes --quiet proj geos hdf4 hdf5 kealib \
-            libnetcdf openjpeg poppler libtiff libpng xerces-c expat libxml2 kealib json-c \
+        conda install --yes --quiet proj geos hdf4 hdf5 kealib highfive \
+            libnetcdf openjpeg poppler libtiff libpng xerces-c expat libxml2 json-c \
             cfitsio freexl geotiff libjpeg-turbo libpq libspatialite libwebp-base pcre pcre2 postgresql \
             sqlite tiledb zstd cryptopp cgal doxygen librttopo openssl liblzma-devel \
             openjdk ant qhull armadillo blas blas-devel libblas libcblas liblapack liblapacke blosc libarchive \

--- a/.github/workflows/cmake_builds.yml
+++ b/.github/workflows/cmake_builds.yml
@@ -436,11 +436,10 @@ jobs:
 
     - name: Install dependency
       shell: bash -l {0}
-      # Disabling "kealib" because of https://github.com/OSGeo/gdal/issues/14857
       run: |
         conda install --yes --quiet curl libiconv icu python=3.12 pip swig numpy "pytest!=9.1.0" pytest-env pytest-benchmark filelock zlib lxml jsonschema setuptools
-        conda install --yes --quiet proj geos hdf4 hdf5 \
-            libnetcdf openjpeg poppler libtiff libpng xerces-c expat libxml2 json-c \
+        conda install --yes --quiet proj geos hdf4 hdf5 kealib \
+            libnetcdf openjpeg poppler libtiff libpng xerces-c expat libxml2 kealib json-c \
             cfitsio freexl geotiff libjpeg-turbo libpq libspatialite libwebp-base pcre pcre2 postgresql \
             sqlite tiledb zstd cryptopp cgal doxygen librttopo openssl liblzma-devel \
             openjdk ant qhull armadillo blas blas-devel libblas libcblas liblapack liblapacke blosc libarchive \

